### DISCUSSION TITLE
Is it possible to try this for Hoower?

### DISCUSSION THREAD

**Discussion by algida** [ofalvai/home-assistant-candy#63] [Device support improvements]
I have Hoover OneFi washing machine HBWMO 916TAHC-S.
Thank you.
Ales

**Comment by ofalvai** [comment]
It's possible, but nothing is guaranteed to work for devices I don't physically have in my home :)

The README has a section about what to do if it turns out your device is not working with this integration.

**Comment by algida** [comment]
Thank you.
Will try it.
Now I have problems even connet device to router, because I have changed router from time when It worked (ssid and phassphrase) are different and It does not work. I cannot find any clue how to reset wifi settings for my device :-(
